src/main/java/com/joy/joyfillandroid/MultiDropDown.kt

```diff
@@ -0,0 +1,224 @@
+package com.joy.joyfillandroid
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.Typeface
+import android.graphics.drawable.GradientDrawable
+import android.os.Build
+import android.text.InputType
+import android.util.AttributeSet
+import android.view.Gravity
+import android.view.KeyEvent
+import android.view.View
+import android.widget.LinearLayout
+import android.widget.TextView
+import androidx.appcompat.widget.AppCompatAutoCompleteTextView
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetDialog
+
+class MultiDropDown@JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : LinearLayout(context, attrs, defStyleAttr) {
+
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var adapter: DropDownAdapter
+    val autoCompleteTextView = AppCompatAutoCompleteTextView(context)
+    private lateinit var bottomSheetDialog: BottomSheetDialog
+    var list= ArrayList<RecyclerModel>()
+
+    init {
+
+        addView(createAutoCompleteTextView(context))
+
+    }
+    fun createAutoCompleteTextView(context: Context): LinearLayout {
+        val linearLayout = LinearLayout(context)
+
+        linearLayout.orientation = LinearLayout.VERTICAL
+        linearLayout.layoutParams = LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT)
+        linearLayout.gravity = Gravity.START
+
+        val textField = TextField(context)
+        textField.text ="MultiDropDown Field"
+        linearLayout.addView(textField)
+        val layoutParams = LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.WRAP_CONTENT)
+
+        val backgroundDrawable = createEditTextBackground()
+
+        autoCompleteTextView.id = R.id.autoTextView
+        autoCompleteTextView.layoutParams = LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.WRAP_CONTENT)
+        autoCompleteTextView.background= backgroundDrawable
+        autoCompleteTextView.hint = "Option 1"
```

Comment:
what is the autoComplete hint? Would that be dynamic or always static as Option 1?

Reply:
It is placeholder hint text that will be dynamic user can pass it according to their need.